themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
+emailVerificationSubject=Ověření e-mailu
+emailVerificationBody=Někdo vytvořil účet {2} s touto e-mailovou adresou. Pokud jste to vy, klikněte na níže uvedený odkaz a ověřte svou e-mailovou adresu \n\n{0}\n\nTento odkaz vyprší za {1} minuty.\n\nPokud jste tento účet nevytvořili, zpráva.
+emailVerificationBodyHtml=<p>Někdo vytvořil účet {2} s touto e-mailovou adresou. Pokud jste to vy, klikněte na níže uvedený odkaz a ověřte svou e-mailovou adresu. </p><p><a href="{0}">Odkaz na ověření e-mailové adresy</a></p><p>Platnost odkazu vyprší za {1} minut.</p><p>Pokud jste tento účet nevytvořili, ignorujte tuto zprávu.</p>
+emailTestSubject=[KEYCLOAK] - testovací zpráva SMTP
+emailTestBody=Toto je testovací zpráva
+emailTestBodyHtml=<p>Toto je testovací zpráva </p>
+identityProviderLinkSubject=Odkaz {0}
+identityProviderLinkBody=Někdo chce propojit váš účet "{1}" s účtem {0}"uživatele {2} . Pokud jste to vy, klikněte na níže uvedený odkaz a propojte účty. \n\n{3}\n\nTento odkaz vyprší za {4} minuty.\n\nPokud nechcete propojit účet, ignorujte tuto zprávu . Pokud propojíte účty, budete se moci přihlásit do {1} až {0}.
+identityProviderLinkBodyHtml=<p> Někdo chce propojit váš účet <b>{1}</b> s účtem <b>{0}</b> uživatele {2} . Pokud jste to vy, kliknutím na odkaz níže propojíte účty </p><p><a href="{3}">Odkaz na potvrzení propojení</a></p><p>Tento odkaz vyprší v rámci {4} minut.</p><p>Pokud nechcete propojit účet, ignorujte tuto zprávu. Pokud propojíte účty, budete se moci přihlásit do {1} až {0}.</p>
+passwordResetSubject=Zrušit heslo
+passwordResetBody=Někdo právě požádal o změnu pověření účtu {2}. Pokud jste to vy, klikněte na odkaz níže, abyste je resetovali.\n\n{0}\n\nTento odkaz a kód uplynou do {1} minut.\n\nPokud nechcete obnovit vaše pověření , ignorujte tuto zprávu a nic se nezmění.
+passwordResetBodyHtml=<p> Někdo právě požádal o změnu pověření vašeho účtu {2}. Pokud jste to vy, klikněte na odkaz níže, abyste je resetovali.</p><p><a href="{0}">Odkaz na obnovení pověření </a></p><p> Platnost tohoto odkazu vyprší během {1} minut.</p><p> Pokud nechcete obnovit vaše pověření, ignorujte tuto zprávu a nic se nezmění.</p>
```

Comment:
`<p>Někdo právě požádal o změnu hesla u vašeho účtu {2}. Pokud jste to vy, pro jeho změnu klikněte na odkaz níže.</p><p><a href="{0}">Odkaz na změnu hesla.</a></p><p>Platnost tohoto odkazu je {3}.</p><p>Pokud heslo změnit nechcete, tuto zprávu ignorujte a nic se nezmění.</p>`